human-viz-bridge/src/iCubEffortBridge.cpp

```diff
@@ -0,0 +1,330 @@
+#include "sensor_msgs_Temperature.h"
+#include "std_msgs_Header.h"
+
+#include <iDynTree/ModelIO/ModelLoader.h>
+#include <iDynTree/Model/Model.h>
+#include <iDynTree/Model/Traversal.h>
+#include <TickTime.h>
+#include <yarp/dev/ControlBoardInterfaces.h>
+#include <yarp/dev/ITorqueControl.h>
+#include <yarp/dev/PolyDriver.h>
+#include <yarp/os/BufferedPort.h>
+#include <yarp/os/LogStream.h>
+#include <yarp/os/Network.h>
+#include <yarp/os/Node.h>
+#include <yarp/os/Property.h>
+#include <yarp/os/Publisher.h>
+#include <yarp/os/RFModule.h>
+#include <yarp/os/Time.h>
+#include <yarp/sig/Vector.h>
+
+#include <iDynTree/Core/EigenHelpers.h>
+
+#include <algorithm>
+#include <iostream>
+#include <limits.h>
+#include <string>
+#include <vector>
+#include <cmath>
+
+using namespace std;
+using namespace yarp::os;
+using namespace yarp::sig;
+using namespace yarp::dev;
+
+//---------------------------------------------------------------------------
+//Utility functions for parsing INI file
+static bool parseStringListOption(const Value &option, vector<string> &parsedList);
+//---------------------------------------------------------------------------
+
+struct EffortPublisher {
+    string linkEffort;
+    vector<size_t> jointDoFs;
+    Publisher<sensor_msgs_Temperature> *publisher;
+    sensor_msgs_Temperature effortMsg;
```

Comment:
If we want to do that we have to put all the assignments for that variable in the update section